src/Rule/Url.php

```diff
@@ -108,6 +101,11 @@ public function pattern(string $pattern): self
     public function enableIDN(): self
     {
         $new = clone $this;
+
+        if (!function_exists('idn_to_ascii')) {
+            throw new \RuntimeException('In order to use IDN validation intl extension must be installed and enabled.');
+        }
```

Comment:
```suggestion
        if (!function_exists('idn_to_ascii')) {
            throw new \RuntimeException('In order to use IDN validation intl extension must be installed and enabled.');
        }

        $new = clone $this;
```
There is no need to clone the instance if `idn_to_ascii` doesn't exist.